alerts/ui/src/components/filters/FilterPills.jsx

```diff
@@ -3,27 +3,141 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from "react"
+import React, { useEffect } from "react"
 
 import { Pill, Stack } from "@cloudoperators/juno-ui-components"
-import { useActiveFilters, useFilterActions } from "../../hooks/useAppStore"
+import {
+  useActiveFilters,
+  useFilterActions,
+  useFilterPills,
+} from "../../hooks/useAppStore"
 
 const FilterPills = () => {
   const activeFilters = useActiveFilters()
-  const { removeActiveFilter } = useFilterActions()
+  const filters = useFilterPills()
+  const { removeActiveFilter, addActiveFilter, setFilterPills } =
+    useFilterActions()
+  // useEffect Hook zur Aktualisierung der Filter
+  useEffect(() => {
+    const pills = (prevFilters) => {
```

Comment:
Creating this new merged object seems very complicated and potentially error prone to me. 

An easier solution might be store the two objects for active and paused filters (where paused filters are a subset of active filters). Then when you render the filter pills you do it by iterating over the active filters, and for each pill you check whether that key:value combination is also present in pausedFilters and then render accordingly. That would make it much easier to understand what is happening, and it doesn't require you to run through the list twice.

Let's have a chat about this.

Reply:
Thats not possible because of the architecture of Alerts. Because the "active" label was not existent before the app handels all filters in this Array as active. This change would lead to a refactor of various components.

Reply:
hmm, I'm not sure I understand the problem you describe. But I've probably not explained well enough what I meant. My proposed solution was something like the following:
1. add a `pauseFilter/unpauseFilter` action to the `filter` store slice that works similar to the `removeActiveFilter/addActiveFilter` actions in that it modifies the pausedFilters state and then calls the `filterItems` action of the `alerts` slice afterwards. 
2. In the `filterItems` action we must then modify the section that checks whether the item is being filtered out by an active filter to also take paused filters into account
3. When rendering the list of filter pills you also need to add the check to figure out if the filter is paused as discussed above.